frog/imports/client/Wiki/index.js

```diff
@@ -52,13 +52,19 @@ export const LearningItem = dataFn.LearningItem;
 
 type WikiCompPropsT = {
   location: *,
-  match: {
+  match?: {
     params: {
       wikiId: string,
-      pageTitle: ?string
+      pageTitle: ?string,
+      instance?: string
     }
   },
-  history: Object
+  history: Object,
+  embedPage?: {
```

Comment:
Make sure to update other instances of this variable

``` suggestion
  embeddedPage?: {
```

Reply:
Made the respective changes